backend/routes/api/seasons.js

```diff
@@ -23,474 +24,490 @@ import {
+  return {
+    model: DateRange,
+    as: "dateRanges",
+    attributes: ["id", "startDate", "endDate"],
+    where: {
+      seasonId,
+    },
+    required: false,
+    order: [["startDate", "ASC"]],
+    include: [
+      {
+        model: DateType,
+        as: "dateType",
+        attributes: ["id", "name"],
+      },
+    ],
+  };
+}
+
+/**
+ * Returns a query part for including a Dateable and its DateRanges.
+ * @param {number} seasonId the ID of the DateRanges' Season
+ * @returns {Object} Sequelize query part for fetching Dateable and its DateRanges
+ */
+function dateableAndDatesQueryPart(seasonId) {
+  return {
+    model: Dateable,
+    as: "dateable",
+    include: [dateRangesQueryPart(seasonId)],
+  };
+}
+
+/**
+ * Returns a query part for including FeatureType details with a Feature.
+ * @returns {Object} Sequelize query part for fetching FeatureType details
+ */
+function featureTypeQueryPart() {
+  return {
+    model: FeatureType,
+    as: "featureType",
+    attributes: ["id", "name"],
+  };
+}
+
+// Common attributes for all Season queries
+const SEASON_ATTRIBUTES = [
+  "id",
+  "operatingYear",
+  "status",
+  "readyToPublish",
+  "editable",
+  "publishableId",
+];
+
+// Get all form data and DateRanges for a Feature Season
 router.get(
-  "/:seasonId",
+  "/feature/:seasonId",
   asyncHandler(async (req, res) => {
-    // @TODO: Reimplement this endpoint to use the publishable model
-    const error = new Error("Not implemented yet");
-
-    error.status = 501;
-    throw error;
-
-    const { seasonId } = req.params;
+    const seasonId = Number(req.params.seasonId);
 
     const seasonModel = await Season.findByPk(seasonId, {
-      attributes: [
-        "id",
-        "operatingYear",
-        "status",
-        "readyToPublish",
-        "editable",
-      ],
+      attributes: SEASON_ATTRIBUTES,
       include: [
+        // Feature details
         {
-          model: FeatureType,
-          as: "featureType",
-          attributes: ["id", "name", "icon"],
-        },
-        {
-          model: Park,
-          as: "park",
-          attributes: ["id", "name", "orcs"],
-        },
-        {
-          model: SeasonChangeLog,
-          as: "changeLogs",
-          attributes: ["id", "notes", "createdAt"],
-          // Filter out empty notes
-          where: {
-            notes: {
-              [Op.ne]: "",
-            },
-          },
-          required: false,
-          order: [["createdAt", "DESC"]],
+          model: Feature,
+          as: "feature",
           include: [
+            featureTypeQueryPart(),
+
+            // Park Area, if any
             {
-              model: User,
-              as: "user",
+              model: ParkArea,
+              as: "parkArea",
               attributes: ["id", "name"],
+              required: false,
+            },
+
+            // Park details
+            {
+              model: Park,
+              as: "park",
             },
```

Comment:
Not sure if we need ParkArea and Park inside Feature since we have a form at each level. But if not, we can edit the model anytime 🙂